cosign/go.mod

```
@@ -0,0 +1,104 @@
+module github.com/ratify-project/ratify-verifier-go/cosign
+
+go 1.23.4
```

It should be `1.23.0` or `1.23.1` as this is a library
```suggestion
go 1.23.0
```

---

Updated to 1.24.0.